pkg/manager/manager.go

```diff
@@ -64,6 +64,15 @@ type Manager interface {
 	// election was configured.
 	Elected() <-chan struct{}
 
+	// AddMetricsExtraHandler adds an extra handler served on path to the http server that serves metrics.
+	// Might be useful to register some diagnostic endpoints e.g. pprof.
+	//
+	// Note that these endpoints meant to be sensitive and shouldn't be exposed publicly.
+	//
+	// If the simple path -> handler mapping offered here is not enough,
+	// a new http server/listener should be added as Runnable to the manager via Add method.
+	AddMetricsExtraHandler(path string, handler http.Handler) error
```

Comment:
Fine for me, but wondering if AddMetricsServerExtraHandler would be better.

If you add a pprof handler it's not really a "metrics handler"